core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -38,8 +38,17 @@ function makeIdNumeric(id) {
 }
 
 function formatDate(date) {
-  if (date <= 0) return "-";
-  else return date.split(".")[0].replace("T", " ");
+  if (date <= 0) {
+    return "-";
```

Comment:
just for curious, what does a single `-` mean here? invalid value?

Reply:
Yes,  invalid value. Maybe we can remove this.